src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3541,10 +3541,10 @@ private BoundNode BindSimpleProgram(CompilationUnitSyntax compilationUnit, Bindi
         {
             var simpleProgram = (SynthesizedSimpleProgramEntryPointSymbol)ContainingMemberOrLambda;
```

Comment:
Unused now?

Reply:
Yup. Fixed that one and a couple of other locations now. Thanks